### ISSUE TITLE
 CGPasses.h.inc No such file (16.0.0 flang) 

### ISSUE THREAD

**Issue by hpcpony** [llvm/llvm-project#61543]
Still an issue from #54128, #57680?

```console
  .
  .
  .
[ 90%] Building CXX object tools/flang/lib/Frontend/CMakeFiles/obj.flangFrontend.dir/CodeGenOptions.cpp.o
[ 90%] Building CXX object tools/flang/lib/Frontend/CMakeFiles/obj.flangFrontend.dir/FrontendAction.cpp.o
[ 90%] Building CXX object tools/flang/lib/Frontend/CMakeFiles/obj.flangFrontend.dir/FrontendActions.cpp.o
In file included from /scratch/llvm-project-16.0.0.src/flang/include/flang/Tools/CLOptions.inc:16,
                 from /scratch/llvm-project-16.0.0.src/flang/lib/Frontend/FrontendActions.cpp:508:
/scratch/llvm-project-16.0.0.src/flang/include/flang/Optimizer/CodeGen/CodeGen.h:27:10: fatal error: flang/Optimizer/CodeGen/CGPasses.h.inc: No such file or directory
   27 | #include "flang/Optimizer/CodeGen/CGPasses.h.inc"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```

**Comment by jschueller** [comment]
I'm also seeing this error with unix Makefiles / -j1

**Comment by jschueller** [comment]
I compiled 16.0.6, but I have a fix now see #71691
